api/openapi.yaml

```diff
@@ -966,8 +982,17 @@ components:
           format: date-time
         scanConfig:
           $ref: '#/components/schemas/ScanConfigRelationship'
-        scanConfigSnapshot:
-          $ref: '#/components/schemas/ScanConfigSnapshot'
+        scanFamiliesConfig:
+          $ref: '#/components/schemas/ScanFamiliesConfig'
+        scope:
+          $ref: '#/components/schemas/ScanScopeType'
+        maxParallelScanners:
+          type: 'integer'
+          minimum: 1
+          maximum: 20
+          description: "The maximum number of scanners that can run in parallel for each scan"
+        scannerInstanceCreationConfig:
+          $ref: '#/components/schemas/ScannerInstanceCreationConfig'
```

Comment:
I think I mentioned this in the VMClarity channel, I was thinking about doing something like:

ScanConfig becomes:
```
schedule:
disabled: 
scanConfig:
    scope:
    assetScanConfig:
        scanFamilyConfig:
        instanceCreationConfig:
        etc...
```

Scan becomes:
```
scope:
assetScanConfig:
    scanFamilyConfig:
    instanceCreationConfig:
```

WDYT? This will allow us to create reusable objects, but its a bigger remodel. Personally I think it would be better like this.

Reply:
ok, we can do it later, just thought since you've now added the same 3/4 fields to different objects we can make them as a separate object.